src/perf_tests/Python.PerformanceTests.csproj

```diff
@@ -3,12 +3,19 @@
   <PropertyGroup>
     <TargetFramework>net461</TargetFramework>
     <Configurations>DebugMono;DebugMonoPY3;ReleaseMono;ReleaseMonoPY3;DebugWin;DebugWinPY3;ReleaseWin;ReleaseWinPY3</Configurations>
+    <OutputPath>bin\</OutputPath>
 
     <IsPackable>false</IsPackable>
+
+    <Platforms>x64;x86</Platforms>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="BenchmarkDotNet" Version="0.11.5" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net461" Version="1.0.0-preview.2">
```

Comment:
Any particular reason for referencing net461 explicitly? I was under the impression that the package selects the correct one without a version.

Reply:
I am unsure if selection happens before downloading or after. If it has to download all packages first, then it should shorten build time. I can look it up, but I hope we will get rid of this soon, as it is only needed for pythonnet-2.3 packages, that don't support .NET Standard.

Reply:
Fixed